### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by killy10o10**
added a new JavaScript file and import it as a module:
It will contain methods related to the status updates
(completed: true/false).
Added event listener to the checkbox change.
Updated the items' object's value the for completed key upon user actions.
Implemented a function for the "Clear all completed" button using the filter() method).
Stored the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

**Comment by killy10o10** [comment]
Before you flag that there are linter errors just know I have no linter errors locally when I run them. I don't really know what is up with Github actions but I have been getting this and there seems to be no solution that has worked so far. You are more than welcome to clone my project for review purposes and any suggestions would be highly appreciated. Thanks in advance.

File: src/modules/addToDo.js [hunk 1]
```diff
@@ -24,6 +26,29 @@ const addToDoList = (todoValue) => {
...
+//       icon.classList.toggle('hide');
+//     });
+//     currenTrashIcon.forEach((trash) => {
+//       trash.classList.toggle('active');
+//     });
+//     const currentCheckBox = e.target.closest('.list-item');
+//     currentCheckBox.classList.toggle('cancelList');
+//   }
+// });
+
```
Comment (OmarRamoun) [inline src/modules/addToDo.js]: - [x] [REQUIRED] Please, remove commented code because it makes the code base not clean.

**Review by OmarRamoun — CHANGES_REQUESTED** [review]
# 🟡 Almost There! 🏃

## Hi @killy10o10 👋

Good job so far! 👏. Your Coding Style is Impressive! 💪.

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃

## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** 🌟 that can improve your code quality 💯 and help you **get a better job** 💼.

_I will guide 📔 you through fixing 🔧 these bugs 🐛, just **follow my comments down below** 🔽 and make sure you **don't miss any** of them, in order to get your project approved ✅_

## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- 🚧 I have check the linters and I believe the problem has to do with the linters themselves and not with your code
- The code is top-notch! 💯 and have a huge potential.

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/addToDo.js (hunk 1 above)
Comment (killy10o10) [reply src/modules/addToDo.js]: @OmarRamoun Thanks

**Review by bimbolabuari — APPROVED** [review]
Hi @killy10o10,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**